css/css-fonts/font-size-adjust-012.html

```diff
@@ -19,7 +19,7 @@
             --secondary-font: 'ahem-ex-250';
         }
         .adjusted {
-            font-size-adjust: 0.5;
```

Comment:
@xiaochengh Are these `font-size-adjust` values intentionally changed in this CL to percentages as well. If yes, why? I understand the issue to be about `size-adjust` values?

Reply:
No. Find-and-replace mistakes...

I'll revert this one.